gnovm/pkg/gnolang/preprocess.go

```diff
@@ -2590,6 +2590,9 @@ func preprocess1(store Store, ctx BlockNode, n Node) Node {
 				//
 				// In short, the relationship between tmp and dst is:
 				//   `type dst tmp`.
+				if n.Name == blankIdentifier {
+					return n, TRANS_CONTINUE
+				}
```

Comment:
i mean we can *also* do the same thing here:
https://github.com/gnolang/gno/blob/499e4ba7e2c8125f3ec81398ef6ff4fa62152a8c/gnovm/pkg/gnolang/preprocess.go#L4800


isn't it?

Reply:
hum this will cause a runtime crash I think. we need alway to define it

Reply:
yes right should be done here https://github.com/gnolang/gno/pull/4367/commits/494220060e5cc5fb8fefd5bc3440d498371c1bd7